src/intelligence_layer/evaluation/dataset/studio_dataset_repository.py

```diff
@@ -188,3 +196,16 @@ def examples(
             if examples_to_skip is not None and data["id"] in examples_to_skip:
                 continue
             yield Example[input_type, expected_output_type].model_validate(data)  # type: ignore
+
+    def map_to_studio_example(
+        self, example_to_map: Example[Input, ExpectedOutput]
+    ) -> StudioExample[Input, ExpectedOutput]:
+        return StudioExample(**example_to_map.model_dump())
+
+    def map_to_many_studio_example(
+        self, examples_to_map: Iterable[Example[Input, ExpectedOutput]]
+    ) -> Iterable[StudioExample[Input, ExpectedOutput]]:
+        return [self.map_to_studio_example(example) for example in examples_to_map]
```

Comment:
() these brackets would be cleaner as it does a generator instead of putting everything into memory